doc/CMakeLists.txt

```diff
@@ -323,7 +323,7 @@ add_doxygen_docset(nrf ${NRF_BASE} ${NRF_VERSION})
 #-------------------------------------------------------------------------------
 # docset: mcuboot
 
-add_docset(mcuboot "2.1.99")
+add_docset(mcuboot "3.1.0")
```

Comment:
what is going on here? It is based on mcuboot 2.2.0-rc2, there isn't even an mcuboot 3.x

Reply:
Please use mcuboot 2.2.0-rc2

Reply:
Updated. Please check.